hyperswitchSDK/Core/PaymentMethodManagement/PaymentMethodMangement.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  PaymentMethodMangement.swift
+//  hyperswitch
+//
+//  Created by Shivam Nan on 15/09/24.
+//
+
+import Foundation
+import WebKit
+
+public class PaymentMethodMangement {
+    @Published var ephemeralKey: String?
```

Comment:
why are we using Published var here

Reply:
this is used for observable objects in swiftUI only

Reply:
true. Made it a private var.